quickstart.js

```diff
@@ -104,6 +105,7 @@ function logResponse(response) {
 async function quickstart() {
   var url;
   var response;
+  var queryParams = {};

```

Comment:
The existing example was to illustrate getting all participants, so we don't want to add the query param to that one.

I think now that you've fixed it, it'll be pretty obvious how to do a call with a query param and we don't need a specific example. But if you want to include one, you should add a new example at the end, after the existing ones. (maybe a participant search by name or something)

Reply:
Removed

Reply:
I think the `queryParams` var is no longer used now and can also be removed.